core/clingutils/src/TClingUtils.cxx

```diff
@@ -549,6 +548,17 @@ bool TClingLookupHelper::ExistingTypeCheck(const std::string &tname,
    else return false;
 }
 
+bool TClingLookupHelper::CheckInClassTable(const std::string &tname, std::string &result)
```

Comment:
Consider adding the `////...` seen before other functions ... and documentation string (which indeed is missing in most other functions :( ).